kansha/board/view.py

```diff
@@ -140,10 +139,10 @@ def render_Board_num_matches(self, h, comp, *args):
     if self.card_matches:
 
         if None in self.card_matches:
```

Comment:
This condition is very strange. Why the above `if self.card_matches` condition is not enough?